nvflare/lighter/impl/cert.py

```diff
@@ -60,6 +60,7 @@ def initialize(self, ctx):
             )
             self.pub_key = self.pri_key.public_key()
             self.subject = self.root_cert.subject
+            self.issuer = self.subject.get_attributes_for_oid(NameOID.COMMON_NAME)[0].value
```

Comment:
Does this always work for every certs?
Or does this assume the certs were created in a specific way?

Reply:
For our persistent state, it is always true.  If it fails, then it's an exception and the provisioning stops because we can not proceed with incorrect root cert.